resources/public/api/conf/1.0/broughtForwardLosses_create.raml

```diff
@@ -0,0 +1,39 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
```

Comment:
Duplicate headers.contentTypeJson... please delete one :)

Reply:
This is your file.